test/utils/server_version.hxx

```diff
@@ -22,12 +22,16 @@
 
 namespace test::utils
 {
+
+enum class server_edition { enterprise, community, unknown };
```

Comment:
could you make `unknown` first? so that it will be default for uninitialized cases